WalletWasabi/Mono/CommandSet.cs

```diff
@@ -403,7 +403,7 @@ private static void ExtractToken(ref string input, out string rest)
 					continue;
 				}
 
-				for (int j = i; j < top; j++)
+				for (int j = i + 1; j < top; j++)
```

Comment:
What is the point of this change?

Reply:
When we set `j = i` we will make an extra (unnecessary) iteration.
Meaning we will check `if (char.IsWhiteSpace(input[j]))` but we know already that it is false because we have already checked it here `if (char.IsWhiteSpace(input[i]))` and if this was true we would not even get to the for j loop.